### DISCUSSION TITLE
VScode extention?

### DISCUSSION THREAD

**Discussion by r-rajaneesh** [pawelsalawa/sqlitestudio#4394] [Ideas]
It would be cool if this was also a VScode extention!

**Comment by pawelsalawa** [comment]
What would this extension bring to VScode?

**Reply by r-rajaneesh** [reply]
you can use this in development, so if any changes are needed to make, we can do it directly in VScode, instead of the app

**Reply by pawelsalawa** [reply]
What changes? What would the extension do?

**Reply by r-rajaneesh** [reply]
See, suppose you are creating something like a discord bot? And you wanted to change a value but you are stuck out, cause the db stored a different value, and there are other keys that are important, and you don't have a backup of the DB. So this will be useful

**Reply by pawelsalawa** [reply]
Okay, but I still don't understand how is this relevant to SQLiteStudio?

**Reply by pawelsalawa** [reply]
I mean if you need to edit database, use SQLiteStudio.

**Reply by r-rajaneesh** [reply]
uhhhh
SQLiteStudio can be made into a VScode extension so devs can use the app as a extension in VScode, for faster development

**Reply by pawelsalawa** [reply]
It cannot be just made into extension. It's completely different application, in many ways. Also I'm not sure how would that be faster than just double-clicking on db file to open it with SQLiteStudio.
